usb-autorunner-dev/usb-autorunner

```diff
@@ -43,11 +57,11 @@ case ${option} in
     config)
         config ${2}
         ;;
-    help) 
+    help)
         help
-        ;; 
-    *)  
-        echo "option '${1}' not found" 
+        ;;
+    *)
+        echo "command '${1}' not found"
```

Comment:
"Command" ?

Reply:
Commands such as: `generate`, `config` and `help`
While Options are arguments for the commands (eg. `-p`, `-c`...)

Reply:
This is just about the upper case "C".